packages/fxa-settings/src/pages/Signin/index.tsx

```diff
@@ -0,0 +1,239 @@
+                    Continue to{' '}
+                    <span>
+                      <PocketLogo
+                        className="inline"
+                        aria-label={RelyingParties.pocket}
+                      />
+                    </span>
+                  </span>
+                </FtlMsg>
+              )}
+              {!isPocketClient && ServiceLogo && (
+                <FtlMsg id="signin-subheader-with-logo">
+                  <span className="card-subheader">
+                    Continue to{' '}
+                    <span>
+                      <ServiceLogo
+                        className="inline"
+                        aria-label={serviceName}
+                      />
+                    </span>
+                  </span>
+                </FtlMsg>
+              )}
+              {!isPocketClient && !ServiceLogo && (
+                <FtlMsg
+                  id="signin-subheader-without-logo"
+                  vars={{ serviceName }}
+                >
+                  <span className="card-subheader">
+                    Continue to {serviceName}
+                  </span>
+                </FtlMsg>
+              )}
+            </>
+          </>
+        )}
+      </div>
+      <section>
+        {/* Alerts and success messages originally went here */}
+        <div className="mt-9">
+          {/* When we get to the functionality stage, we can probably replace this with the Avatar component in Settings*/}
+          <div className="mx-auto h-24 w-24 mobileLandscape:h-40 mobileLandscape:w-40"></div>
+          <div className="my-5 text-base break-all">{email}</div>
+        </div>
+        <form noValidate {...{ onSubmit }}>
+          <input type="email" className="email hidden" value={email} disabled />
+
+          {isPasswordNeeded && (
+            <InputPassword
+              anchorStart
+              className="mb-5 text-start"
+              label={localizedPasswordFormLabel}
+              hasErrors={error.length > 0}
+              errorText={error}
+              tooltipPosition="bottom"
+              required
+              autoFocus
+              onChange={(e) => setPassword(e.currentTarget.value)}
+            />
+          )}
+          {/* This non-fulfilled input tricks the browser, when trying to
+              sign in with the wrong password, into not showing the doorhanger.
+           */}
+          <input className="hidden" required />
+
+          <div className="flex">
+            <FtlMsg id="signin-button">
+              <button className="cta-primary cta-xl" type="submit">
+                Sign in
+              </button>
+            </FtlMsg>
+          </div>
+        </form>
+        <div className="text-grey-500 my-5 text-xs">
+          {isPocketClient ? (
+            <FtlMsg id="firefox-and-pocket-tos">
+              <p className="text-xs">
+                By proceeding, you agree to:
+                <ul>
```

Comment:
It's invalid HTML to place a `ul` inside a `p` tag. 😩 I think if you just close `</p>` before the UL, that should be fine for FTL too?